packages/plugins/plugin-space/src/components/MembersContainer.tsx

```diff
@@ -118,37 +124,47 @@ export const MembersContainer = ({
 
   return (
     <Clipboard.Provider>
-      <StackItem.Content classNames='p-2 block overflow-y-auto'>
-        <ControlSection
-          title={t('members verbose label', { ns: SPACE_PLUGIN })}
-          description={t('members description', { ns: SPACE_PLUGIN })}
-        >
+      <StackItem.Content classNames='block overflow-y-auto'>
```

Comment:
If we’re doing this a lot maybe this needs a flag like `blockLayout` or something…

Reply:
we should review all of these settings-type articles once things settle